product_listing.css

```diff
@@ -0,0 +1,200 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  --primary-color: #03045e;
+  --text-color: white;
+  --black-color: black;
+  --hover-btn-color: #caf0f8;
```

Comment:
You can declare color variables inside :root

Reply:
okay, done